### PR TITLE
Create missing starter files and create separate finished files for each lesson 

### PR DESCRIPTION
**Pull request by Asjas**
This is a PR to update the starter files and solution files for the course lessons.

### CODE REVIEW

File: playground/modules/currencies.js [hunk 1]
```diff
@@ -1,38 +0,0 @@
-const currencies = {
```
Comment (wesbos) [inline playground/modules/currencies.js:1]: Can we keep this one? It's not something we type

File: playground/modules/base.css [hunk 2]
```diff
@@ -1,116 +0,0 @@
...
-
```
Comment (wesbos) [inline playground/modules/base.css:2]: Keep this file too - it's the base CSS for the exercises. They don't type it themselves

File: exercises/16 - Debugging/debugging-FINISHED.js [hunk 3]
```diff
@@ -7,10 +7,14 @@ const people = [
 /* Victor Mono */
```
Comment (wesbos) [inline exercises/16 - Debugging/debugging-FINISHED.js:7]: can you take these 2 comments out too

Comment (Asjas) [reply exercises/16 - Debugging/debugging-FINISHED.js]: i removed these 🙂

**Pull request merged**